aiofiles/__init__.py

```diff
@@ -1,6 +1,7 @@
 """Utilities for asyncio-friendly file handling."""
 from .threadpool import open
-
+from .aiotempfile import NamedAioTemporaryFile, AioTemporaryFile
```

Comment:
`aio` prefix is too redundant IMHO.
Just `aiofiles.tempfile` and  `NamedTemporaryFile`/`TemporaryFile` would be enough.